### PR TITLE
Added Data Schema Based on diagram

### PR DESCRIPTION
**Pull request by Shaheryar0054**
- Added schema data based on diagram
- Join tables from many-to-many relationships
- Added the FK indexes.
clinic_diagram

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ALTER TABLE
+    "medical_histories" ADD PRIMARY KEY("id");
+CREATE TABLE "invoice_items"(
+    "id" INTEGER NOT NULL,
+    "unit_price" DECIMAL(8, 2) NOT NULL,
+    "quantity" INTEGER NOT NULL,
+    "total_price" DECIMAL(8, 2) NOT NULL,
+    "invoice_id" INTEGER NOT NULL,
+    "treatment_id" INTEGER NOT NULL
+);
```
Comment (DeliceLydia) [inline schema_based_on_diagram.sql]: - [x] Please create a table between `medical_histories` and `treatments` 
```sql
 CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ALTER TABLE
+    "invoice_items" ADD CONSTRAINT "invoice_items_treatment_id_foreign" FOREIGN KEY("treatment_id") REFERENCES "treatments"("id");
+ALTER TABLE
+    "treatments" ADD CONSTRAINT "treatments_id_foreign" FOREIGN KEY("id") REFERENCES "medical_histories"("id");
+ALTER TABLE
+    "invoice_items" ADD CONSTRAINT "invoice_items_invoice_id_foreign" FOREIGN KEY("invoice_id") REFERENCES "invoices"("id");
+ALTER TABLE
+    "medical_histories" ADD CONSTRAINT "medical_histories_admitted_at_foreign" FOREIGN KEY("admitted_at") REFERENCES "patients"("id");
+ALTER TABLE
+    "medical_histories" ADD CONSTRAINT "medical_histories_id_foreign" FOREIGN KEY("id") REFERENCES "treatments"("id");
```
Comment (DeliceLydia) [inline schema_based_on_diagram.sql]: - [x] Please create indexes on all FOREIGN KEYS 
ex:
`CREATE INDEX ON medical_histories (patient_id)`
etc

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Created all tables required
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 2 above)
Comment (tamoorsaeed22) [reply schema_based_on_diagram.sql]: @DeliceLydia we have updated the requirements as demanded and created indexes on all FOREIGN KEYS

**Review by uwadonat — APPROVED** [review]
## Hi @Shaheryar0054,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights 
- Good commit history ✔️
- Wel-organized and clean project ✔️
appr

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**